openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx

```diff
@@ -499,11 +492,10 @@ const EntityPageInfo = ({
               </div>
             ) : null}
             <Button
-              className="tw-rounded tw-mb-1 tw-flex bg-[#D9CEEE] tw-ml-2"
+              className="tw-rounded tw-mb-1 tw-flex bg-[#D9CEEE] tw-ml-2 tw-border tw-border-primary"
```

Comment:
Not sure why we need this class if it's imported form antD

Reply:
@chirag-madlani , This is for adding an outline to the button.